### PR TITLE
Added examples to DateAndTime::Calculations [ci skip]

### PR DESCRIPTION
**Pull request by piscolomo**

### CODE REVIEW

File: activesupport/lib/active_support/core_ext/date_and_time/calculations.rb [hunk 1]
```diff
@@ -92,15 +92,28 @@ def years_since(years)
     end
 
     # Returns a new date/time at the start of the month.
+    #
+    #   today = Date.today # => Thu, 18 Jun 2015
+    #   today.beginning_of_month # => Mon, 01 Jun 2015
+    #
     # DateTime objects will have a time set to 0:00.
+    #
+    #   today = DateTime.today # => Thu, 18 Jun 2015 15:23:13 UTC +00:00
```
Comment (repinel) [inline activesupport/lib/active_support/core_ext/date_and_time/calculations.rb]: `DateTime` doesn't respond to `today`. Use `current`.

(applies to all `DateTime.today` occurrences)

Comment (TheBlasfem) [inline activesupport/lib/active_support/core_ext/date_and_time/calculations.rb]: thanks for the observation @repinel I hadn't noticed it, I updated the file with `Time.current` similar to the documentation of `DateAndTime::Calculations#next_week` (more below)

**Comment by repinel** [comment]
Looks good. Thanks

**Comment by sgrif** [comment]
The docs mention `DateTime`, but the examples use `Time`. Can we reference the correct object here?

**Comment by TheBlasfem** [comment]
sure @sgrif I've made that change now, the examples now are using `DateTime`

**Comment by sgrif** [comment]
:heart:

**Pull request merged**